test/parallel/test-path.js

```diff
@@ -406,6 +422,11 @@ assert.strictEqual(path.win32.normalize('a//b//.'), 'a\\b');
 assert.strictEqual(path.win32.normalize('//server/share/dir/file.ext'),
                    '\\\\server\\share\\dir\\file.ext');
 assert.strictEqual(path.win32.normalize('/a/b/c/../../../x/y/z'), '\\x\\y\\z');
```

Comment:
I'd rather leave that to a dedicated PR, as there are many other tests that do not use this pattern in the file.